containers/daap-presigned-url/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/ministryofjustice/data-platform-daap-python-base:0.3.0
+FROM ghcr.io/ministryofjustice/data-platform-daap-python-base:0.5.0
```

Comment:
Could you bump this to `1.0.0` please

Reply:
were we gonna pin this at `:latest` for some images while we're developing?